### ISSUE TITLE
Display Freezes or Chrome on Host Freezes when Using GPU-PV

### ISSUE THREAD

**Issue by FlyGuyGo** [jamesstringerparsec/Easy-GPU-PV#235]
I have an issue where the display on my host will lockup/freeze when using a VM with GPU-PV. Sometimes this issue only affects Chrome on the host but the rest of the interface will not be frozen. 

This can be temporarily fixed by resetting the GPU driver by pressing Windows Key + Shift + Ctrl + B. However, it seems to happen frequently enough to where it is too disruptive. 

I have a 5600x with a 3070 Ti and 16GB of ram. I do not overclock.

I have tried the following to fix this issue:
Fresh GPU Driver Install with DDU
Updated Chipset
Disabling + Reenabling Hyper V
Fresh OS install
Disable resize bar
Use studio GPU drivers

I am curious to here if this has happened to anyone else or if anyone knows of a fix for this.

**Comment by baesik** [comment]
Hey, just wanted to add a comment that this happens to me. Mouse cursor does move, so it's not a complete lockup. Thanks for reminding me of that GPU shortcut command!

**Comment by FlyGuyGo** [comment]
> ursor does move, so it's not a complete lockup. Thanks for reminding me of that GPU shortcut command!

The shortcut helps big time for sure! What is the configuration of your machine? Just so we can keep track.

**Comment by Zsr123zsr123** [comment]
Also happens to me. Mouse does move, keyboard no responses. Entire user session is freeze.
Before I can use RDP on anther device to sign in another account, so the freeze session will back to lock screen, and I can sign in back. Recently, this trick does not work, it will stuck at waiting for response for the account that is correctly logged or just welcome...... in on RDP.

**Comment by techgo** [comment]
> Also happens to me. Mouse does move, keyboard no responses. Entire user session is freeze. Before I can use RDP on anther device to sign in another account, so the freeze session will back to lock screen, and I can sign in back. Recently, this trick does not work, it will stuck at waiting for response for the account that is correctly logged in or just welcome...... on RDP side.

Have you tried using "Windows Key + Shift + Ctrl + B" to recover your screen when this issue happens?

**Comment by frosty3907** [comment]
Yep same issue here but only with Chrome on host machine, freezes up.
I have been working around it by opening Edge, which for some reason gets chrome to start responding again.

**Comment by Zsr123zsr123** [comment]
> > Also happens to me. Mouse does move, keyboard no responses. Entire user session is freeze. Before I can use RDP on anther device to sign in another account, so the freeze session will back to lock screen, and I can sign in back. Recently, this trick does not work, it will stuck at waiting for response for the account that is correctly logged in or just welcome...... on RDP side.
> 
> Have you tried using "Windows Key + Shift + Ctrl + B" to recover your screen when this issue happens?

Hello, no I have not. Since I saw this could be solved by this resetting GPU, I reduced GPU Resource Allocation on a VM from 100% down to 75%. I had this issue every day on last week, but I have not got the issue after I reduced it. That VM runs some GPU memory consuming softwares.

**Comment by Zsr123zsr123** [comment]
> > > Also happens to me. Mouse does move, keyboard no responses. Entire user session is freeze. Before I can use RDP on anther device to sign in another account, so the freeze session will back to lock screen, and I can sign in back. Recently, this trick does not work, it will stuck at waiting for response for the account that is correctly logged in or just welcome...... on RDP side.
> > 
> > 
> > Have you tried using "Windows Key + Shift + Ctrl + B" to recover your screen when this issue happens?
> 
> Hello, no I have not. Since I saw this could be solved by this resetting GPU, I reduced GPU Resource Allocation on a VM from 100% down to 75%. I had this issue every day on last week, but I have not got the issue after I reduced it. That VM runs some GPU memory consuming softwares.

Damn, I just had that issue, Windows Key + Shift + Ctrl + B did work.

**Comment by techgo** [comment]
> Chrome maximized with GPU-PV startup will cause a screen freeze. I turn off the monitor's power and turn it on after a few seconds, and the screen will return to normal. It seems like a conflict with Chrome's GPU acceleration.

I agree that a conflict with Chrome is the culprit.

**Comment by frosty3907** [comment]
> Chrome maximized with GPU-PV startup will cause a screen freeze. I turn off the monitor's power and turn it on after a few seconds, and the screen will return to normal. It seems like a conflict with Chrome's GPU acceleration.

Chrome being maximized is not the issue, I have this issue and don't run it maximized.
